packages/react/src/components/List/List.jsx

```diff
@@ -184,7 +184,7 @@ const List = forwardRef((props, ref) => {
           editingStyle={editingStyle}
           secondaryValue={secondaryValue}
           rowActions={rowActions}
-          onSelect={editingStyle ? () => {} : () => handleSelect(item.id)}
+          onSelect={editingStyle ? () => {} : () => handleSelect(item.id, parentId)}
```

Comment:
This seems unrelated to the rest of these changes. Was this intentional?

Reply:
This was a fix for select function. In some place we are sending parentid back and some place we aren't.  So it's best to send this information so that we can use it in a callback function.